doc/sphinx/source/recipes/recipe_monitor.rst

```diff
@@ -189,6 +189,18 @@ Timeseries of tas including a reference dataset.
 
 Annual cycle of tas including a reference dataset.
 
+.. _fig_diurnal_cycle:
+.. figure::  /recipes/figures/monitor/diurnalcycle_pr_tropics_EC-Earth3_3hr_historical_r1i1p1f1.png
+   :align:   center
+   :width:   14cm
```

Comment:
Please add a figure caption

Reply:
Done. I also indented all figure captions so they show up correctly in the pdf.